ZFSin/zfs/lib/libzfs/libzfs_import.c

```diff
@@ -1833,7 +1836,7 @@ zpool_find_import_impl(libzfs_handle_t *hdl, importargs_t *iarg)
 			slice->rn_name = zfs_strdup(hdl, name);
 			slice->rn_parent = zfs_strdup(hdl, path);
 			slice->rn_avl = &slice_cache;
-			slice->rn_dfd = dfd;
+			slice->rn_dfd = HTOI(dfd);
```

Comment:
This one isn't a `_win()` specific version, so it shouldn't use `HTOI()` - so it should probably be renamed to `_win()` in future (or rather, this will live in the `.../os/windows/` location in the future repository.